subprojects/dependency-management/src/testFixtures/groovy/org/gradle/api/internal/artifacts/verification/DependencyVerificationFixture.groovy

```diff
@@ -88,11 +88,15 @@ class DependencyVerificationFixture {
 
     void assertXmlContents(String expected) {
         def actualContents = TextUtil.normaliseLineSeparators(verificationFile.text)
+        // remove namespace declaration for readability of tests
+        actualContents = actualContents.replaceAll("<verification-metadata .+>", "<verification-metadata>")
```

Comment:
Did you mean `actualContents.replaceAll("<verification-metadata [^>]+>", "<verification-metadata>")` ?

Reply:
In theory, yes but in practice here it's not required (it's not multiline matching).